tools/bridge/bridge/config.py

```diff
@@ -47,6 +47,8 @@ def validate_checksum_address(address: Any) -> None:
 OPTIONAL_CONFIG_ENTRIES_WITH_DEFAULTS: Dict[str, Any] = {
     "foreign_chain_max_reorg_depth": 10,
     "transfer_event_poll_interval": 5,
+    "foreign_chain_event_fetch_start_block_number": 0,
```

Comment:
That's a really declarative name

Reply:
`transfer_event_fetch_start_block_number` would be a *little* bit shorter and more consistent with `transfer_event_poll_interval`.

Reply:
For the moment okay, but I guess we need to generalize this in future anyways. Or would you introduce such option per event? We must differ at least between both chains.

Reply:
Here we go. As stated I would avoid to go for event specific configuration (the polling is bad enough, overseen when it becomes obsolete). Therefore this configuration refers to the following information:
 - relates to the `foreign_`- / `home_chain` (we need to differ at least between these both)
 - this is about the `block_number`(s)
 - describes where to `start` (kinda essential for the meaning)
 - is used for the event fetching

I would argue that the usage does not matter that much. It could be also used for different purpose in future than fetching event (even I don't see so far and probably never will). Therefore `foreign_chain_start_block_number` would be an alternative without loosing "important" information. This is 10 characters less and even 6 less than @jannikluhn his suggestion. 
@saschagoebel @jannikluhn are you fine with this name? :upside_down_face:

Reply:
We can even discuss if we strike out `chain` from all config options and hope it is intuitive.

Reply:
I made a "proposal" commit. I'll rewrite it if I get feedback from you. Else please feel free to close the conversation and merge this PR.

Reply:
`foreign_chain_start_block_number` sounds a bit weird (start of the foreign chain?), but I don't think it matters too much. Changing it later should be trivial if we want to.

Reply:
Okay, since we fail to come up with a better solution and the old version is the most descriptive one without loosing information, I go for the `Changing it later should be trivial if we want to.`.